### PR TITLE
Template for new selection algo height

### PR DESCRIPTION
**Pull request by 0xKrishna**

### CODE REVIEW

File: mainnet.toml [hunk 1]
```diff
@@ -0,0 +1 @@
+new_selection_algo_height = 375300
```
Comment (jdkanani) [inline mainnet.toml:1]: Can you add a new line at the of the file? https://stackoverflow.com/questions/44704968/visual-studio-code-insert-new-line-at-the-end-of-files

File: bor/keeper.go [hunk 2]
```diff
@@ -232,7 +232,7 @@ func (k *Keeper) SelectNextProducers(ctx sdk.Context, seed common.Hash) (vals []
 	// TODO remove old selection algorigthm
 	// select next producers using seed as blockheader hash
 	fn := SelectNextProducers
-	if ctx.BlockHeight() < 375300 {
+	if ctx.BlockHeight() < helper.NewSelectionAlgoHeight {
```
Comment (jdkanani) [inline bor/keeper.go:235]: Fix github actions

**Review by jdkanani — CHANGES_REQUESTED** [review]

File: mainnet.toml (hunk 1 above)
Comment (0xKrishna) [reply mainnet.toml]: Done

File: bor/keeper.go (hunk 2 above)
Comment (0xKrishna) [reply bor/keeper.go:235]: Done

**Review by ssandeep — APPROVED** [review]

**Review by jdkanani — APPROVED** [review]

**Pull request merged**